src/content/docs/guides/upgrade-migrate/from-tauri-1.mdx

```diff
@@ -1057,3 +1057,41 @@ tauri::Builder::default()
         Ok(())
   })
 ```
+
+## Prepare for Mobile
```

Comment:
@lucasfernog do you think it makes sense to have this section moved all the way to the very top of the page? Assuming that `tauri migrate` isn't going to make any of these changes and that they're safe to do before running that command?

Reply:
I think we shouldn't do these changes automatically because it can be tricky on projects that aren't following the template we have. And yeah we could move it to the top, I just wanted it separate from the other steps.